spec/ad-services/live-connect/live-connect.spec.ts

```diff
@@ -53,6 +54,30 @@ describe('LiveConnect', () => {
 		expect(loadScriptStub.called).to.equal(false);
 	});
 
+	it('Live Connect is called with default qf if feature flag is not set', async () => {
+		window.liQ = { resolve: noop };
+		const liqStub = global.sandbox.stub(window.liQ, 'resolve').returns(Promise.resolve({} as any));
+
+		await liveConnect.resolveAndReportId('unifiedId', 'fandom', { qf: 0.3 });
+
+		expect(
+			liqStub.args[0].filter((arg) => typeof arg === 'object').map((arg) => JSON.stringify(arg)),
+		).to.contain('{"qf":0.3}');
+	});
+
+	it('Live Connect is called with qf value from feature flag', async () => {
+		window.liQ = { resolve: noop };
+		const liqStub = global.sandbox.stub(window.liQ, 'resolve').returns(Promise.resolve({} as any));
+
+		instantConfigStub.get.withArgs('icLiveConnectQf').returns(0.5);
+
+		await liveConnect.resolveAndReportId('unifiedId', 'fandom', { qf: 0.3 });
+
+		expect(
+			liqStub.args[0].filter((arg) => typeof arg === 'object').map((arg) => JSON.stringify(arg)),
```

Comment:
```suggestion
			liqStub.calledWith(sinon.match.func, sinon.match.func, { qf: 0.5 })
```
looks better and provides better assertion